tasks/test_results_processor.py

```diff
@@ -143,12 +146,105 @@ def _bulk_write_tests_to_db(
                     )
                 )
 
+                def update_daily_total():
+                    daily_totals[test_id]["last_duration_seconds"] = duration_seconds
```

Comment:
Nit: change these to daily_totals.get(test_id, {}).get("last_duration_seconds") to prevent null errors